2025-06-18 09:51:16,725 - [INFO] - from play in pool-18-thread-6 
Listening for HTTP on /0:0:0:0:0:0:0:0:8089

2025-06-18 09:51:52,617 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 09:51:54,455 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2025-06-18 10:09:53,549 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:09:53,940 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:14:11,315 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:14:11,578 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:14:44,448 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:14:44,665 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:15:13,057 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:15:13,227 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:15:54,318 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:15:54,526 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:16:17,065 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:16:17,302 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:17:05,003 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:17:05,810 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:20:34,629 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:20:34,809 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:22:42,039 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:22:42,214 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:23:27,465 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:23:27,671 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:23:43,162 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:23:43,370 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:24:05,450 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:24:05,696 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:25:58,310 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:25:58,492 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:31:13,063 - [ERROR] - from application in New I/O worker #6 


! @89nlf24m5 - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[not found: type director]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:33:49,441 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:33:50,051 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:38:11,791 - [ERROR] - from application in New I/O worker #11 


! @89nlg0094 - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[value id is not a member of (java.util.Map[String,Int], java.util.List[String], java.util.List[String], java.util.List[String], java.util.ArrayList[Array[String]], java.util.ArrayList[Array[String]], models.Ctacorreo, java.util.List[models.Director], Integer, Integer) => play.twirl.api.HtmlFormat.Appendable]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:39:24,431 - [ERROR] - from application in New I/O worker #11 


! @89nlg43kj - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[value getId is not a member of (java.util.Map[String,Int], java.util.List[String], java.util.List[String], java.util.List[String], java.util.ArrayList[Array[String]], java.util.ArrayList[Array[String]], models.Ctacorreo, java.util.List[models.Director], Integer, Integer) => play.twirl.api.HtmlFormat.Appendable]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:41:23,927 - [ERROR] - from application in New I/O worker #11 


! @89nlgaof0 - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value getId]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:41:38,372 - [ERROR] - from application in New I/O worker #11 


! @89nlgbjog - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value id]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:41:51,987 - [ERROR] - from application in New I/O worker #11 


! @89nlgce26 - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[value getId is not a member of object scala.collection.immutable.::]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:44:18,769 - [ERROR] - from application in New I/O worker #11 


! @89nlgkn5i - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[value getId is not a member of (java.util.Map[String,Int], java.util.List[String], java.util.List[String], java.util.List[String], java.util.ArrayList[Array[String]], java.util.ArrayList[Array[String]], models.Ctacorreo, java.util.List[models.Director], Integer, Integer) => play.twirl.api.HtmlFormat.Appendable]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:46:52,471 - [ERROR] - from application in New I/O worker #11 


! @89nlh3gf4 - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[value getId is not a member of (java.util.Map[String,Int], java.util.List[String], java.util.List[String], java.util.List[String], java.util.ArrayList[Array[String]], java.util.ArrayList[Array[String]], models.Ctacorreo, java.util.List[models.Director], Integer, Integer) => play.twirl.api.HtmlFormat.Appendable]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:47:42,785 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:47:43,742 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:48:58,512 - [ERROR] - from application in New I/O worker #13 


! @89nlhal10 - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[value id is not a member of (java.util.Map[String,Int], java.util.List[String], java.util.List[String], java.util.List[String], java.util.ArrayList[Array[String]], java.util.ArrayList[Array[String]], models.Ctacorreo, java.util.List[models.Director], Integer, Integer) => play.twirl.api.HtmlFormat.Appendable]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:49:22,274 - [ERROR] - from application in New I/O worker #13 


! @89nlhc44n - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[missing arguments for method -> in class ArrowAssoc;
follow this method with `_' if you want to treat it as a partially applied function]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:52:54,618 - [ERROR] - from application in New I/O worker #13 


! @89nlho67o - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[object models.Director is not a value]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:53:06,161 - [ERROR] - from application in New I/O worker #13 


! @89nlhon9p - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[object models.Director is not a value]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 10:56:55,901 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:56:56,341 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:57:18,903 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:57:19,156 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 10:58:14,601 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 10:58:14,837 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:00:46,087 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:00:46,370 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:06:38,219 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:06:39,106 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:07:25,626 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:07:25,846 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:07:48,397 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:07:49,048 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:08:06,740 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:08:06,991 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:09:22,308 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:09:22,546 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:10:43,147 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:10:43,383 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:11:45,600 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:11:45,870 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:11:54,598 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:11:54,846 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:12:20,225 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:12:20,436 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:15:27,785 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:15:28,244 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:15:58,774 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:15:59,114 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:18:13,316 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:18:13,701 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:18:52,745 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:18:53,063 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:19:43,060 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:19:43,334 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:22:50,280 - [ERROR] - from application in New I/O worker #13 


! @89nllmafp - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value directorEMS]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 11:27:22,086 - [ERROR] - from application in New I/O worker #13 


! @89nlmbmi1 - Internal server error, for (GET) [/admin] ->

play.PlayExceptions$CompilationException: Compilation error[error: cannot find symbol]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:235) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$play$PlayReloader$$anon$$taskFailureHandler$1.apply(PlayReloader.scala:230) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.play$PlayReloader$$anon$$taskFailureHandler(PlayReloader.scala:230) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:90) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.4.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:90) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.7.jar:2.3.7]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.7.jar:2.3.7]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.4.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.4.jar:na]

2025-06-18 11:27:57,339 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:27:57,737 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:29:59,899 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:30:00,202 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:31:02,463 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:31:02,787 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:33:24,363 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:33:24,629 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:35:24,335 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:35:24,539 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:37:28,986 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:37:29,503 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:47:25,252 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:47:25,749 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:49:12,287 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:49:13,006 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 11:50:26,190 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 11:50:26,537 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:18:23,393 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:18:23,673 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:19:48,219 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:19:48,491 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:20:01,839 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:20:02,081 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:20:39,143 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:20:39,319 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:22:58,490 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:22:59,118 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:23:39,872 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:23:40,021 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:24:19,032 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:24:19,356 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:25:56,878 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:25:57,781 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:26:24,403 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:26:24,572 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2025-06-18 12:26:54,273 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:26:54,436 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:27:09,683 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:27:09,829 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:28:52,615 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:28:52,787 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:29:13,505 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:29:13,673 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:30:11,125 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:30:11,263 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:30:29,848 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:30:30,025 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:33:46,601 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:33:47,228 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:34:50,856 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:34:51,929 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:36:48,611 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:36:48,768 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:39:47,041 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:39:47,976 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:40:01,893 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:40:02,061 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:40:19,179 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:40:19,347 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:41:00,779 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:41:00,924 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:41:49,090 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:41:49,430 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:43:10,092 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:43:10,272 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:43:34,074 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:43:34,793 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:44:06,054 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:44:06,462 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:44:44,126 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:44:44,274 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:45:27,338 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:45:27,500 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:45:50,654 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:45:50,925 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:46:08,422 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:46:08,611 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:46:24,382 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:46:24,523 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:46:35,372 - [INFO] - from play in play-internal-execution-context-16 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:46:35,544 - [INFO] - from play in play-internal-execution-context-16 
Application started (Dev)

2025-06-18 12:49:18,945 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:49:19,117 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:50:06,888 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:50:08,326 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:50:43,504 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:50:43,692 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:51:43,516 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:51:43,852 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:53:47,010 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:53:47,344 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:57:46,426 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:57:46,856 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 12:59:19,332 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 12:59:19,599 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:00:48,307 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:00:48,557 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:01:30,213 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:01:30,426 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:01:54,692 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:01:54,836 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:03:25,549 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:03:25,731 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:03:44,477 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:03:44,859 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:06:11,994 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:06:12,274 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:11:04,485 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:11:04,809 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:24:19,560 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:24:19,848 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:26:31,001 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:26:31,266 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:39:50,428 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:39:50,741 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 13:40:34,498 - [INFO] - from play in play-internal-execution-context-14 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 13:40:34,656 - [INFO] - from play in play-internal-execution-context-14 
Application started (Dev)

2025-06-18 14:22:18,729 - [INFO] - from play in play-internal-execution-context-18 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 14:22:19,018 - [INFO] - from play in play-internal-execution-context-18 
Application started (Dev)

2025-06-18 14:22:51,871 - [INFO] - from play in play-internal-execution-context-18 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 14:22:52,156 - [INFO] - from play in play-internal-execution-context-18 
Application started (Dev)

2025-06-18 14:23:47,529 - [INFO] - from play in play-internal-execution-context-18 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 14:23:47,882 - [INFO] - from play in play-internal-execution-context-18 
Application started (Dev)

2025-06-18 14:24:29,709 - [INFO] - from play in play-internal-execution-context-18 
database [default] connected at jdbc:mysql://localhost:3306/erdd_dev7

2025-06-18 14:24:30,225 - [INFO] - from play in play-internal-execution-context-18 
Application started (Dev)

